ballerina/Module.md

```diff
@@ -242,14 +242,15 @@ sql:ParameterizedQuery query = `SELECT * FROM students
 stream<Student, sql:Error?> resultStream = dbClient->query(query);
 
 // Iterating the returned table.
-error? e = resultStream.forEach(function(Student student) {
-   //Can perform operations using the record 'student' of type `Student`.
-});
+error? e = check from Student student in resultStream
```

Comment:
Why is this returning an error and using check at the same time?

Reply:
Fixed in 96f6de7